src/main/java/pl/pandziak/ticketbookingapp/configuration/DataProvider.java

```diff
@@ -0,0 +1,69 @@
+package pl.pandziak.ticketbookingapp.configuration;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.event.ContextRefreshedEvent;
+import org.springframework.context.event.EventListener;
+import pl.pandziak.ticketbookingapp.domain.screening.ScreeningFacade;
+import pl.pandziak.ticketbookingapp.domain.screening.dtos.ScreeningRequestDto;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Service providing testing data to in-memory database.
+ */
+@Configuration
+class DataProvider {
+
+    private final Logger logger = LoggerFactory.getLogger(DataProvider.class);
+
+    private static final String CSV_FILE_SEPARATOR = ";";
+    private static final String DATA_FILE_NAME = "data.csv";
+    private static final String DATE_PATTERN = "yyyy-MM-dd HH:mm";
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern(DATE_PATTERN);
+
+    private final ScreeningFacade screeningFacade;
+
+    DataProvider(ScreeningFacade screeningFacade) {
+        this.screeningFacade = screeningFacade;
+    }
+
+    @EventListener
+    public void populateInMemoryDatabase(ContextRefreshedEvent ctx) {
+        try (InputStream inputStream = getClass().getClassLoader().getResourceAsStream(DATA_FILE_NAME);
```

Comment:
jest już masa utilsów na to :)

Reply:
To było najprostsze rozwiązanie jakie w tamtym momencie udało mi się znaleźć. Ale na przyszłość poszukam innego.